packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -77,21 +79,23 @@ library ChainAssertions {
 
     /// @notice Asserts that the L1CrossDomainMessenger is setup correctly
     function checkL1CrossDomainMessenger(Types.ContractSet memory _contracts, Vm _vm, bool _isProxy) internal view {
+        console.log("Running chain assertions on the L1CrossDomainMessenger");
         L1CrossDomainMessenger messenger = L1CrossDomainMessenger(_contracts.L1CrossDomainMessenger);
 
         require(address(messenger.portal()) == _contracts.OptimismPortal);
         require(address(messenger.PORTAL()) == _contracts.OptimismPortal);
         if (_isProxy) {
-            require(address(messenger.superchainConfig()) == _contracts.SuperchainConfig);
-            bytes32 xdmSenderSlot = _vm.load(address(messenger), bytes32(uint256(204)));
-            require(address(uint160(uint256(xdmSenderSlot))) == Constants.DEFAULT_L2_SENDER);
+            // require(address(messenger.superchainConfig()) == _contracts.SuperchainConfig);
+            // bytes32 xdmSenderSlot = _vm.load(address(messenger), bytes32(uint256(204)));
+            // require(address(uint160(uint256(xdmSenderSlot))) == Constants.DEFAULT_L2_SENDER);
         } else {
-            require(address(messenger.superchainConfig()) == address(0));
+            // require(address(messenger.superchainConfig()) == address(0));
```

Comment:
Intentionally left commented out? Should reintroduce or remove these, but it looks like we need them still.